### ISSUE TITLE
Issue in `get_order_history_crypto()`

### ISSUE THREAD

**Issue by amandeepfj** [JestonBlu/RobinHood#129]
`get_order_history_crypto()` returns wrong value for `price` column. I have attached 4 orders as examples.

![image](https://user-images.githubusercontent.com/941995/120055567-010e0e80-c005-11eb-8745-36d8a61262d4.png)
![image](https://user-images.githubusercontent.com/941995/120055585-14b97500-c005-11eb-91d2-3bb95f658f17.png)
![image](https://user-images.githubusercontent.com/941995/120055614-3f0b3280-c005-11eb-86d3-045149f2b85f.png)
![image](https://user-images.githubusercontent.com/941995/120055485-75947d80-c004-11eb-92b2-ab206be1eebf.png)

Here is the dataframe which is returned. I have added a last column to show time in EDT.
Code used to generate below dataframe:
```
get_order_history_crypto(RH) %>% as_tibble() %>% 
    mutate(created_at_in_EDT = as_datetime(created_at, tz = "US/Eastern"))
```
![image](https://user-images.githubusercontent.com/941995/120055553-ecca1180-c004-11eb-871a-6b4f212651f8.png)

**Comment by JestonBlu** [comment]
I have not been able to reproduce your issue. When i look at my own order history it matches what I am getting out of the API. I have limited history though. The API only returns a single price column and im not rounding or operating on it at all so I dont know why it would be different. I will keep looking but right now nothing obvious is sticking out to me.

**Comment by JestonBlu** [comment]
So while I was looking into another issue I realized that it was similar to your question. Check out the linked issue to see an example of me going through a crypto order. #132 

I also did some googling on the difference between the total notional and the filled amount. The total notional is what you are actually charged, and one forum i found suggest the the filled amount being slightly different is a known bug.
